app/views/searches/_form.html.erb

```diff
@@ -2,6 +2,9 @@
   <div class="form-group">
     <label class="form-label" for="query">
       <span class="heading-small">Roles, skills and interests</span>
+      <% if local_assigns[:show_hints] %>
+        <span class="form-hint">e.g. <span class="example">Retail</span></span>
```

Comment:
class js-hidden will load in a more graceful manor when you first hit the page.  Can display the full html tag in some cases when JS turned off.